.prettierrc.json

```diff
@@ -2,14 +2,24 @@
   "singleQuote": true,
   "printWidth": 120,
   "semi": true,
-  "tabWidth": 2,
+  "tabWidth": 4,
   "trailingComma": "all",
   "bracketSpacing": true,
   "bracketSameLine": false,
   "arrowParens": "avoid",
-  "endOfLine": "lf",
+  "endOfLine": "auto",
```

Comment:
why did you change prettier configs? it will be effected on everywhere

Reply:
this 'lf' causes problem for files on mac. 
But I will reverse it. I usually keep it on my machine.